lib/nodes/fpga.cpp

```diff
@@ -209,7 +209,7 @@ int FpgaNode::fastWrite(Sample *smps[], unsigned cnt) {
 
   assert(cnt == 1 && smps != nullptr && smps[0] != nullptr);
 
-  auto mem = MemoryAccessor<uint32_t>(*blockTx);
+  auto mem = MemoryAccessor<float>(*blockTx);
```

Comment:
this is not enough to fix your issue. See https://github.com/VILLASframework/node/commit/323c77b58ffd2bcd33370ea917252e8c32cb900d